packages/terra-html-table/src/terra-dev-site/doc/html-table/Table.1.doc.mdx

```diff
@@ -0,0 +1,66 @@
+import { Badge } from 'terra-html-table/package.json?dev-site-package';
+
+import DefaultTable from '../example/DefaultTable?dev-site-example';
+import NoStripedTable from '../example/NoStripedTable?dev-site-example';
+import CompactPaddingTable from '../example/CompactPaddingTable?dev-site-example';
+import StandardPaddingTable from '../example/StandardPaddingTable?dev-site-example';
+import TableWithLongContent from '../example/TableWithLongContent?dev-site-example';
+import TableWithCustomCells from '../example/TableWithCustomCells?dev-site-example';
+
+import TablePropsTable from 'terra-html-table/src/Table?dev-site-props-table';
+import TableHeaderPropsTable from 'terra-html-table/src/TableHeader?dev-site-props-table';
+import TableHeaderCellPropsTable from 'terra-html-table/src/TableHeaderCell?dev-site-props-table';
+import TableRowPropsTable from 'terra-html-table/src/TableRow?dev-site-props-table';
+import TableCellPropsTable from 'terra-html-table/src/TableCell?dev-site-props-table';
+
+<Badge />
+
+# Terra HTML Table
+
+The Terra HTML Table is a component for creating read-only, non-responsive, standard semantic tables. If an interactive table is required, please refer to the terra-table.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-html-table`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+## Peer Dependencies
+
+This component requires the following peer dependencies be installed in your app for the component to properly function.
+
+| Peer Dependency | Version |
+|-|-|
+| react | ^16.8.5 |
+| react-dom | ^16.8.5 |
+
+<!-- AUTO-GENERATED-CONTENT:END -->
```

Comment:
The usage section is missing for the quick reference to the component import(s)

Reply:
updated here https://github.com/cerner/terra-core/pull/2850/commits/1c7c026fa2a75274daed84048871c0613d76d22a.